cmake/OpenMCConfig.cmake.in

```diff
@@ -39,6 +39,6 @@ if(@OPENMC_USE_MCPL@)
   find_package(MCPL REQUIRED)
 endif()
 
-if(@OPENMC_USE_UWUW@)
-  find_package(UWUW REQUIRED)
-endif()
+if(@OPENMC_USE_UWUW@ AND NOT @DAGMC_BUILD_UWUW@)
```

Comment:
`DAGMC_BUILD_UWUW` is a variable read in from the `DAGMCConfig.cmake` file as opposed to a configuration value for OpenMC's CMake to populate here.

```suggestion
if(@OPENMC_USE_UWUW@ AND NOT ${DAGMC_BUILD_UWUW})
```